velox/exec/tests/utils/HiveConnectorTestBase.cpp

```diff
@@ -152,6 +152,38 @@ HiveConnectorTestBase::makeHiveConnectorSplit(
       .build();
 }
 
+// static
+std::shared_ptr<connector::hive::HiveInsertTableHandle>
+HiveConnectorTestBase::makeHiveInsertTableHandle(
+    const std::vector<std::string>& tableColumnNames,
+    const std::vector<TypePtr>& tableColumnTypes,
```

Comment:
Hi @mbasmanova do you think we can combine the two arguments into RowType here? I just feel RowType is a concept for exchanged data. Is it appropriate to be used for table schema?

Reply:
Yes, RowType is appropriate for describing table schema.